tests/models/store.py

```diff
@@ -233,8 +236,11 @@ def test_update_set_last_sync_revision(ru_update_set_last_sync_revision_po):
 
 @pytest.mark.django_db
 def test_update_upload_defaults(en_tutorial_po, system):
-    _update_from_upload_file(en_tutorial_po,
-                             "tests/data/po/tutorial/en/tutorial_update.po")
+    update_store(
+        en_tutorial_po,
+        [("Hello, world", "Hello, world UPDATED")],
```

Comment:
Since we now only pass the units, I think we are not properly testing reading the uploaded file revision. Is it not possible to pass here the full uploaded file content to avoid IO delay, and thus keep using old code?